code/modules/client/preference/preferences_mysql.dm

```diff
@@ -132,6 +133,7 @@
 	// Let's not have this explode if you sneeze on the DB
 	var/DBQuery/query = GLOB.dbcon.NewQuery({"SELECT
 					OOC_Notes,
+					slot,
```

Comment:
This currently breaks the database
![image](https://user-images.githubusercontent.com/25063394/87342159-b2699a00-c542-11ea-94e7-156280f450fc.png)